### ISSUE TITLE
Inline radio buttons are vertically misaligned.

### ISSUE THREAD

**Issue by AlexYursha** [twbs/bootstrap#18582]
I'm experiencing the issue with Bootstrap 3.3.6 in Chrome 47 on OS X El Capitan, but I believe it affects all the environments.

I'm trying to implement a layout where an editable entity is represented as a row, which is a common case, I believe. The problem is that inline radio buttons are misaligned with the rest of the row.

Here is the plunker that demonstrates that inline radio buttons are vertically misaligned with the rest of the row. https://jsfiddle.net/Lacz7z5j/3/

**Comment by twbs-lmvtfy** [comment]
Hi @AlexYursha!

You appear to have posted a live example (https://fiddle.jshell.net/Lacz7z5j/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 39, column 1: [E020](https://github.com/twbs/bootlint/wiki/E020): `.radio-inline` should only be used on `<label>` elements
* line 40, column 1: [E020](https://github.com/twbs/bootlint/wiki/E020): `.radio-inline` should only be used on `<label>` elements
* line 41, column 1: [E020](https://github.com/twbs/bootlint/wiki/E020): `.radio-inline` should only be used on `<label>` elements
* line 39, column 1: [E020](https://github.com/twbs/bootlint/wiki/E020): Incorrect markup used with the `.radio-inline` class. The correct markup structure is label.radio-inline>input[type="radio"]
* line 40, column 1: [E020](https://github.com/twbs/bootlint/wiki/E020): Incorrect markup used with the `.radio-inline` class. The correct markup structure is label.radio-inline>input[type="radio"]
* line 41, column 1: [E020](https://github.com/twbs/bootlint/wiki/E020): Incorrect markup used with the `.radio-inline` class. The correct markup structure is label.radio-inline>input[type="radio"]

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by AlexYursha** [comment]
I'm experiencing the issue with Bootstrap 3.3.6 in Chrome 47 on OS X El Capitan, but I believe it affects all the environments.

I'm trying to implement a layout where an editable entity is represented as a row, which is a common case, I believe. The problem is that inline radio buttons are misaligned with the rest of the row.

Here is the plunker that demonstrates that inline radio buttons are vertically misaligned with the rest of the row. https://jsfiddle.net/Lacz7z5j/3/

**Comment by twbs-lmvtfy** [comment]
Hi @AlexYursha!

You appear to have posted a live example (https://fiddle.jshell.net/Lacz7z5j/3/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 38, column 5: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s
* line 41, column 5: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by AlexYursha** [comment]
I'm experiencing the issue with Bootstrap 3.3.6 in Chrome 47 on OS X El Capitan, but I believe it affects all the environments.

I'm trying to implement a layout where an editable entity is represented as a row, which is a common case, I believe. The problem is that inline radio buttons are misaligned with the rest of the row.

Here is the plunker that demonstrates that inline radio buttons are vertically misaligned with the rest of the row. https://jsfiddle.net/Lacz7z5j/4/

**Comment by RyanZim** [comment]
@cvrebert shouldn't the `awaiting reply` label be removed?

**Comment by AlexYursha** [comment]
No, because the reply is still waited for.

**Comment by cvrebert** [comment]
@RyanZim Yes, indeed.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
